theme/errorList.less

```diff
@@ -117,4 +113,35 @@
     &:hover {
         background-color: @errorListMessageHoverColor;
     }
+}
+
+.inverted-theme .errorList {
+    .errorListHeader {
+        color: @errorListInvertedTextColor;
+    }
+
+    .errorListInner {
+        .exceptionMessage {
+            color: @errorListInvertedTextColor;
+        }
+        .ui.selection.list {
+            color: @errorListInvertedTextColor;
+            cursor: pointer;
```

Comment:
do we need this in the non-inverted theme as well?

Reply:
same question for the border-radius/padding styles below

Reply:
Oops! I forgot to commit removing these, copy / paste error when grabbing the selectors.